metriq_gym/cli.py

```diff
@@ -26,6 +26,14 @@ def parse_arguments() -> argparse.Namespace:
     parser.add_argument(
         "-t", "--trials", type=int, default=8, help="Number of trials (default is 8)"
     )
+    parser.add_argument(
+        "-a",
+        "--action",
+        type=str,
+        choices=["dispatch", "poll"],
```

Comment:
nice that you included this change as well! 
I think I prefer a positional argument for this, that is, `python run.py dispatch <file> --provider ...`
The reason is that I see the keyword arguments more as 'optional' or something that has a default, but here you have to specify it. This is not a strong opinion, but mostly gut feeling from using other cli tools.

Reply:
Given the change in b5cbdc2, we can now do: 

```sh
python metriq_gym/run.py dispatch metriq_gym/schemas/examples/clops.example.json --backend ibm_sherbrooke --provider ibmq
```

And similarly, for polling:

```sh
python metriq_gym/run.py poll --jobs_file .metriq_gym_jobs.jsonl --job_id <JOB_ID>
```

where `<JOB_ID>` is the metriq-gym-assigned job ID in the `*.jsonl` file that is generated after dispatching.